archlinuxcn-mirrorlist

```diff
@@ -1,115 +1,160 @@
 ## 重庆邮电大学 (重庆) (ipv4, http, https)
+## frequency: 1d
 # Server = https://mirrors.cqupt.edu.cn/archlinuxcn/$arch
 
 ## 兰州大学 (甘肃兰州) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirror.lzu.edu.cn/archlinuxcn/$arch
 
 ## 南方科技大学 (广东深圳) (ipv4, ipv6, http, https)
+## frequency: 12h
 # Server = https://mirrors.sustech.edu.cn/archlinuxcn/$arch
 
 ## 哈尔滨工业大学 (黑龙江哈尔滨) (ipv4, ipv6, http, https)
+## frequency: 30m
 # Server = https://mirrors.hit.edu.cn/archlinuxcn/$arch
 
 ## 南阳理工学院 (河南南阳) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirror.nyist.edu.cn/archlinuxcn/$arch
 
 ## 荆楚理工学院 (湖北荆门) (ipv4, https)
+## frequency: 12h
 # Server = https://mirrors.jcut.edu.cn/archlinuxcn/$arch
 
 ## 武昌首义学院 (湖北武汉) (ipv4, ipv6, http, https)
+## frequency: 3m
 # Server = https://mirrors.wsyu.edu.cn/archlinuxcn/$arch
 
 ## 南京大学 (江苏南京) (ipv4, ipv6, http, https)
+## frequency: 1d
 # Server = https://mirrors.nju.edu.cn/archlinuxcn/$arch
 
 ## 南京工业大学 (江苏南京) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirrors.njtech.edu.cn/archlinuxcn/$arch
 
 ## 吉林大学 (吉林长春) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirrors.jlu.edu.cn/archlinuxcn/$arch
 
 ## 沈阳航空航天大学 (辽宁沈阳) (ipv4, ipv6, https)
+## frequency: 2h
 # Server = https://mirrors.sau.edu.cn/archlinuxcn/$arch
 
 ## 西安交通大学 (陕西西安) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirrors.xjtu.edu.cn/archlinuxcn/$arch
 
 ## 上海交通大学 (上海) (ipv4, ipv6, https)
+## frequency: 2h
 # Server = https://mirror.sjtu.edu.cn/archlinux-cn/$arch
 
 ## 上海科技大学 (上海) (ipv4, https)
+## frequency: 7h
 # Server = https://mirrors.shanghaitech.edu.cn/archlinuxcn/$arch
 
-## 网易 (浙江杭州) (ipv4, http, https)
-# Server = https://mirrors.163.com/archlinux-cn/$arch
-
 ## 浙江大学 (浙江杭州) (ipv4, ipv6, http, https)
+## frequency: 6h
 # Server = https://mirrors.zju.edu.cn/archlinuxcn/$arch
 
+## 阿里云 (Global CDN) (ipv4, ipv6, http, https)
+## frequency: 1d
+# Server = https://mirrors.aliyun.com/archlinuxcn/$arch
+
+## 腾讯云 (Global CDN) (ipv4, http, https)
+## frequency: 8h
+# Server = https://mirrors.cloud.tencent.com/archlinuxcn/$arch
+
+## 网易 (浙江杭州) (ipv4, http, https)
+## frequency: 12h
+# Server = https://mirrors.163.com/archlinux-cn/$arch
+
 ## xTom (香港伺服器) (香港南區) (ipv4, ipv6, http, https)
+## Blocking users in mainland China
+## frequency: 1h
 # Server = https://mirrors.xtom.hk/archlinuxcn/$arch
 
 ## 國立成功大學 (臺灣臺南) (ipv4, http, https)
+## frequency: 1h
 # Server = https://archlinux.ccns.ncku.edu.tw/archlinuxcn/$arch
 
 ## xTom (Australia server) (Sydney, Australia) (ipv4, ipv6, http, https)
+## Blocking users in mainland China
+## frequency: 4h
 # Server = https://mirrors.xtom.au/archlinuxcn/$arch
 
 ## xTom (Estonia server) (Tallinn, Estonia) (ipv4, ipv6, http, https)
+## Blocking users in mainland China
+## frequency: 4h
 # Server = https://mirrors.xtom.ee/archlinuxcn/$arch
 
 ## xTom (Germany server) (Düsseldorf, Germany) (ipv4, ipv6, http, https)
+## Blocking users in mainland China
```

Comment:
This site doesn't block CN users.

Reply:
https://github.com/archlinuxcn/mirrorlist-repo/commit/5c6995fb02da8c7157fbd1a7bdd3de81fde5dff2

@oldherl